internal/schema/type.go

```diff
@@ -82,3 +82,21 @@ func (t *Type) GetTags() string {
 
 	return jsonTag + "\"`"
 }
+
+// IsGoType is used to determine if a type in NerdGraph is already a native type of Golang.
+func (t *Type) IsGoType() bool {
+	goTypes := []string{
+		"int",
+		"string",
```

Comment:
This might make a better `map[string]string`, then add other basic types from [go](https://tour.golang.org/basics/11) with aliases too

```go
  goTypes := map[string]string{
    "int":     "int",
    "string":  "string",
    "bool":    "bool",
    "boolean": "bool",
  }
```

Reply:
Perhaps, but I'd like to on that until I have a use case, unless you know if one off hand?

Reply:
GraphQL: `Boolean`, Go: `bool`

To make things more complicated, this should probably be:

`nativeTypes["boolean"]["go"]["bool"]`

So we can generate something beyond Go in the future.  OR have the templating system do the native type conversions in some way.

Reply:
Something like what you've proposed seems reasonable.  Can we take that in another PR?